src/ui/common/src/utils/reactflow.ts

```diff
@@ -121,14 +126,16 @@ async function getPositions(
   }
 }
 
-export const getDagLayoutElements = async (
+export const getDagLayoutElements = (
```

Comment:
Shouldn't we also move this function to a redux thunk as well?

That way we can use the state that we fetched throughout our application.

We can have getDagLayoutElements and getSelectDagPosition either as two separate calls, or just one.